packages/openapi-v3/test/unit/controller-spec/controller-decorators/param-decorators/param.test.ts

```diff
@@ -166,25 +143,26 @@ describe('Routing metadata for parameters', () => {
         .withOperationName('greet')
         .withParameter({
           name: 'names',
-          type: 'array',
-          items: {
-            type: 'string',
+          schema: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
           },
           in: 'query',
         })
         .build();
 
       expect(actualSpec.paths['/greet']['get']).to.eql(expectedSpec);
     });
-
     it('reports error if an array parameter type is not Array', () => {
       expect.throws(
         () => {
           // tslint:disable-next-line:no-unused-variable
           class MyController {
             @get('/greet')
             greet(
-              @param.array('names', 'query', 'string')
+              @param.array('names', 'query', {type: 'string'})
```

Comment:
Is it possible to preserve the old API flavour where the type could be specified as a string?

```ts
@param.array('names', 'query', 'string')
// but also
@param.array('numbers', 'query', {type: 'numer', format: 'int32'});
```

Reply:
@bajtos I opened an issue for it, let's keep discussion under it:
https://github.com/strongloop/loopback-next/issues/1053